ambry-router/src/test/java/com/github/ambry/router/MockKeyManagementService.java

```diff
@@ -30,11 +31,12 @@
   }
 
   @Override
-  public SecretKeySpec getKey(short accountId, short containerId) throws GeneralSecurityException {
+  public SecretKeySpec getKey(RestRequest restRequest, short accountId, short containerId)
```

Comment:
Could we make one of the router tests pass in a non-null restRequest in the options and validate that it gets passed correctly to the KMS impl